third_party/sl4a/README

```diff
@@ -1,4 +1,4 @@
 This folder a heavily modified version of sl4a which acts as an RPC library for
-third-party facades.
+third-party code snippets (facades).
```

Comment:
Should we though? It's kind of like explaining something new with something else that people don't know.
I don't think we should assume our users already are already familiar with SL4A, especially externally.
We could briefly refer to SL4A for background, but I wouldn't try to explain Snippet terms with SL4A terms since new users don't necessarily know SL4A and they shouldn't have to learn SL4A terms before they could use snippet.

Thoughts?